CMakeLists.txt

```diff
@@ -270,6 +269,9 @@ target_include_directories(uv
   PRIVATE
     $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src>)
 target_link_libraries(uv ${uv_libraries})
+if(CMAKE_SYSTEM_NAME STREQUAL "OS390")
+  target_link_libraries(uv -Wl,xplink)
+endif()
```

Comment:
It might be nicer to add `-Wl,xplink` to `uv_libraries` and `uv_test_libraries` instead of duplicating the logic 4 times.

Reply:
Updated.